cfgov/jinja2/v1/_includes/atoms/date-picker.html

```diff
@@ -0,0 +1,61 @@
+{# ==========================================================================
+
+   common_form_fields.date_picker()
+
+   ==========================================================================
+
+   Description: Returns a date picker.
+
+   legend:      Name of the field. Defaults to 'Enter Date Range'.
+
+   required:    Whether the field is required. Defaults to false.
+
+   disabled:    Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+{% if not value %}
+    {% set lipsumText = lipsum(1, false, 10, 25) | safe %}
+    {% do global_dict.update({
+        'date_picker': {
+            'legend':       lipsumText,
+            'required':     true,
+            'disabled':     false
+        }
+    }) %}
+    {% set value = global_dict.date_picker %}
+{% endif %}
+<fieldset class="u-reset">
+    <div class="form-l_col
+                form-l_col-1
+                u-mb0">
+        <legend class="form-label-header">
+            {{ value.legend if value.legend else 'Enter date range' }}
+        </legend>
+    </div>
+    <div class="form-l_col form-l_col-1-2">
+        <label class="micro-copy micro-copy__large"
+               for="form_date-picker-starting-date">
+            From:
+        </label>
+        <input class="input__long"
+               id="form_date-picker-starting-date"
+               placeholder="mm/dd/yyyy"
+               type="text"
+               {{ 'required' if value.required else '' }}
+               {{ 'disabled' if value.disabled else '' }}>
+    </div>
+    <div class="form-l_col
+                form-l_col-1-2
+                input__long">
```

Comment:
That's from the old code, there's no more need for that class and it's been removed.